### PR TITLE
Make the compatible app message less confusing for multi versions apps

### LINKED ISSUE #25364: Confusing app upgrade message on server upgrade availability screen
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. See an available minor upgrade being available (applies to major upgrades likewise)
2. Read a message, that "All apps have an update for this version available"
3. Open the list "Apps with available updates"

image

image

### Expected behaviour
* All my installed apps are listed under "Apps with available updates", because the message said, "ALL" apps would have an update
* When going to the app store, the listed apps actually have an update available

### Actual behaviour
* The list shows only a selection of my apps
* The app store shows only a sub-set of the above list to have updates available at all

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #25364

### CODE REVIEW

File: apps/updatenotification/src/components/UpdateNotification.vue [hunk 1]
```diff
@@ -197,8 +198,8 @@ export default {
 			}
 
 			return this.missingAppUpdates.length === 0
-				? t('updatenotification', '<strong>All</strong> apps have an update for this version available', this)
-				: n('updatenotification', '<strong>%n</strong> app has no update for this version available', '<strong>%n</strong> apps have no update for this version available', this.missingAppUpdates.length)
+				? t('updatenotification', '<strong>All</strong> apps have a compatible version for this Nextcloud version available', this)
+				: n('updatenotification', '<strong>%n</strong> app has no compatible version for this Nextcloud version available', '<strong>%n</strong> apps have no compatible version for this Nextcloud version available', this.missingAppUpdates.length)
```
Comment (PVince81) [inline apps/updatenotification/src/components/UpdateNotification.vue:202]: should we replace Nextcloud with $productName for white labelling ?

**Review by PVince81 — APPROVED** [review]
:+1: but see comment about white labelling

**Review by skjnldsv — APPROVED** [review]

File: apps/updatenotification/src/components/UpdateNotification.vue (hunk 1 above)
Comment (nickvergessen) [reply apps/updatenotification/src/components/UpdateNotification.vue:202]: Since this is admin only I don't think it's worth the troubles

**Pull request merged**